### ISSUE TITLE
Individual start and end time control

### ISSUE THREAD

**Issue by britkat1980** [dewet22/givenergy-modbus#30]
Could v1.0 allow individual setting of start and end times for timeslots rather than just both together.
This is useful for automations and manual control in HA through GivTCP.
It would prevent some race conditions when trying to set a single start or end slot.

**Comment by dewet22** [comment]
@britkat1980 I was wondering if this is still pertinent... My assumption was always that you'd want to always set the slot as a whole, but you've had some time to see what users actually want/use. I assume it is easier because HA only models a single timeslot?

**Comment by britkat1980** [comment]
Yes, this is still important as HA entities can only handle a start or end, not a slot.
